fe1-web/parts/Wallet.tsx

```diff
@@ -0,0 +1,185 @@
+  const [tokensRecovered, setTokensRecovered] = useState(false);
+
+  /* all the below are temporarily used for demonstration purposes (showing seed in UI) */
+  const [showHideSeedTitle, setShowHideSeedTitle] = useState(STRINGS.show_seed_title);
+  const [visibleSeed, setVisibleSeed] = useState(false);
+  const [recoveredSeed, setRecoveredSeed] = useState('');
+  // eslint-disable-next-line max-len
+  const [showHideEncryptedSeedTitle, setShowHideEncryptedSeedTitle] = useState(STRINGS.show_encrypted_seed_title);
+  const [visibleEncryptedSeed, setVisibleEncryptedSeed] = useState(false);
+  const [recoveredEncryptedSeed, setRecoveredEncryptedSeed] = useState('');
+
+  function getStartWalletDisplay() {
+    return (
+      <View style={styleContainer.centered}>
+        <TextBlock bold text={STRINGS.welcome_to_wallet_display} />
+        <TextBlock text={' '} />
+        <TextBlock text={STRINGS.info_to_set_wallet} />
+        <TextBlock text={' '} />
+        <TextBlock text={STRINGS.caution_information_on_seed} />
+        <TextBlock text={' '} />
+        <TextBlock text={' '} />
+        <WideButtonView
+          title={STRINGS.create_new_wallet_button}
+          onPress={() => {
+            const generatedSeed = HDWallet.getNewGeneratedMnemonicSeed();
+            console.log(generatedSeed);
+            setSeed(generatedSeed);
+            setWalletState(STRINGS.wallet_state_show_seed);
+          }}
+        />
+        <WideButtonView
+          title={STRINGS.import_seed_button}
+          onPress={() => {
+            setWalletState(STRINGS.wallet_state_set_seed);
+          }}
+        />
+      </View>
+    );
+  }
+
+  function getInsertSeedWalletDisplay() {
+    return (
+      <View style={styleContainer.centered}>
+        <TextBlock text={STRINGS.type_seed_info} />
+        <TextBlock text={' '} />
+        <TextInput
+          style={styles.textInput}
+          placeholder={STRINGS.type_seed_example}
+          onChangeText={(input: string) => setSeed(input)}
+        />
+        <TextBlock text={' '} />
+        <WideButtonView
+          title={STRINGS.setup_wallet}
+          onPress={() => wallet.initialize(seed)
+            .then((seedIsValid) => ((seedIsValid)
+              ? setWalletState(STRINGS.wallet_state_initialized)
+              : setWalletState(STRINGS.wallet_state_null)))}
+        />
+        <WideButtonView
+          title={STRINGS.back_to_wallet_home}
+          onPress={() => setWalletState(STRINGS.wallet_state_null)}
+        />
+      </View>
+    );
+  }
+
+  function getShowSeedWalletDisplay() {
+    return (
+      <View style={styleContainer.centered}>
+        <TextBlock bold text={STRINGS.show_seed_info} />
+        <TextBlock text={' '} />
+        <TextBlock text={seed} />
+        <TextBlock text={' '} />
+        <WideButtonView
+          title={STRINGS.back_to_wallet_home}
+          onPress={() => setWalletState(STRINGS.wallet_state_null)}
+        />
+      </View>
```

Comment:
This would be a cool feature! However I noticed in already implemented digital wallets that there should be no way for the user to copy the mnemonic, or more generally: **there should be no way for the user to avoid writing down the mnemonic on a piece of paper**. This really helps wallet access-loss issues given by a user which, in a hurry, decides to copy-paste the mnemonic without writing it down.